pkg/compiler/codegen.go

```diff
@@ -238,6 +238,29 @@ func (c *codegen) emitStoreStructField(i int) {
 	emit.Opcodes(c.prog.BinWriter, opcode.ROT, opcode.SETITEM)
 }
 
+func (c *codegen) emitStoreSelectorExpr(n *ast.SelectorExpr) {
```

Comment:
Add documentation, add a comment that this method assumes that right-hand statements are already walked by this moment.